tools/test-ci.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd `dirname $0`/.. > /dev/null
+if [ -z ${PYTHON+x} ]; then export PYTHON=`which python`; fi
```

Comment:
stole from http://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash